modules/latex.php

```diff
@@ -17,8 +17,160 @@
  *
  * $latex e^{\i \pi} + 1 = 0$  ->  [latex]e^{\i \pi} + 1 = 0[/latex]
  * $latex [a, b]$              ->  [latex][a, b][/latex]
+ *
+ * In addition to supporting old markup, we also now support MathJax math delimiters:
```

Comment:
What if someone has `\( ... \)` in their posts already (perhaps a post about LaTeX)?  This will break their content.